README.md

```diff
@@ -243,6 +243,7 @@ rules:
         - team-1
       users:
         - user-1
+      rank: 2
```

Comment:
minFellowRank: 2 ?
why is this in allowedToSkipRule? means that if I'm rank: 3 then no reviews needed?

Reply:
Oh, I put it in the wrong place by accident. I fixed it now.